core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -395,14 +378,13 @@ var TopicEditorPage = function() {
 
   this.navigateToTopicEditorTab = async function() {
     var topicEditorTab = element(by.css('.protractor-test-edit-topic-tab'));
-    await waitFor.elementToBeClickable(
-      topicEditorTab, 'Topic editor tab taking too long to be clickable');
-    await topicEditorTab.click();
+    await action.click(
+      'Topic Editor Tab', topicEditorTab);
```

Comment:
This fits on one line now, doesn't it? (I haven't counted characters; I'm just guessing)

Reply:
yes, done